web-common/src/features/dashboards/selectors.ts

```diff
@@ -22,6 +23,7 @@ export function useDashboard(instanceId: string, metricViewName: string) {
 }
 
 export function useValidDashboards(instanceId: string) {
+  // This is used in cloud as well so do not use "useClientFilteredResources"
```

Comment:
**Issue**: The comment suggests not using 'useClientFilteredResources' due to its use in cloud environments, but the code has been changed to use it. <br> **Fix**: Revert to 'useFilteredResources' or ensure that 'useClientFilteredResources' can operate correctly in cloud environments. <br> **Code Suggestion**: 
 ```
 +  // This is used in cloud as well so do not use "useClientFilteredResources"
 -  return useClientFilteredResources(instanceId, ResourceKind.MetricsView, (data) =>
 +  return useFilteredResources(instanceId, ResourceKind.MetricsView, (data) =>
 ```